internal/common/common.go

```diff
@@ -48,6 +48,8 @@ const (
 	FamilyIPv6 int32 = 6
 
 	DefaultCPUArchitecture = "x86_64"
+	X86CPUArchitecture     = DefaultCPUArchitecture
```

Comment:
I think that this making more sense 
```suggestion
	X86CPUArchitecture = "x86_64"
	DefaultCPUArchitecture = X86CPUArchitecture
```

Reply:
done